src/parser/smt2/smt2.h

```diff
@@ -425,6 +425,17 @@ class Smt2 : public Parser
       }
     }
   }
+  /** Set named attribute
+   *
+   * This is called when expression expr is annotated with a name, i.e.
+   * (! expr :named sexpr). It sets up the necessary information to process
+   * this naming, including marking that expr is the last named term.
+   *
+   * We construct an expression symbol whose name is the name of s-expression
+   * which is used later for tracking assertions in unsat cores. This
```

Comment:
```suggestion
   * sexpr, which is used later for tracking assertions in unsat cores. This
```

The name of s-expression you are referring to is the argument `sexpr` right?

Reply:
No, it is a symbol we construct whose name is the same as the `sexpr` (at least that's how its currently implemented).